### ISSUE TITLE
"来園者" needs to be replaced with "ゲスト"

### ISSUE THREAD

**Issue by vrimori** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#715]
Strongly recommend to replace "来園者" with "ゲスト" because, in the paragraph of [Azure Active Directory] in this page, I can see "来園者" is used as a localized Japanese word of "guest" in English, however, it should be "ゲスト" which stands for guest user of Microsoft Teams.

---
#### ドキュメントの詳細

⚠ *このセクションを編集しないでください。 docs.microsoft.com で必須です ➟ GitHub の問題のリンク。*

* ID: 2b05e642-c4e5-f1b5-5315-d3ef1690a0cd
* Version Independent ID: 01b7e0b5-3e50-3de0-e3e9-aeeeda43da12
* Content: [Microsoft Teams でのゲスト アクセスを承認する](https://docs.microsoft.com/ja-jp/microsoftteams/teams-dependencies#feedback)
* Content Source: [Teams/Teams-dependencies.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness-pr.ja-JP/blob/live-sxs/Teams/Teams-dependencies.md)
* Service: **msteams**
* GitHub Login: @laithalshamri
* Microsoft Alias: **laal**

**Comment by jambirk** [comment]
@samanthaw Please Look into this.

**Comment by AndreaBarr** [comment]
@vrimori Thank you for this feedback.

We appriciacte you taking the time to make these documents better. I am getting this over to the Teams team so we can get this addressed. 

I was able to get this into the backlog for the Teams team. The team will get this prioritized into a sprint plan. Thank you for reporting and making the docs better. Much appreciated.

I made a note to request the team to update this when the work is complete.
